server/src/routes/api/v1/bookBookListRouter.js

```diff
@@ -0,0 +1,53 @@
+import express from "express";
```

Comment:
why is all this logic in a `bookBooklistRouter`? Im not following. all of these endpoints should be in your `bookListRouter`